frontend/src/views/Layout/MainLayout.vue

```diff
@@ -121,7 +121,7 @@ export default class MainLayout extends Vue {
 
   getMenuItems() {
     if (this.userClaims) {
-      return filterRoutesByClaims(this.routes, this.userClaims);
+      return filterRoutesByClaims()(this.routes, this.userClaims);
```

Comment:
Here also, let's just export `getMenuItems` from the router and use it here. Let the router filter by claims and meta itself.